src/app/components/cds/media-cards/Card.js

```diff
@@ -31,7 +31,7 @@ const Card = ({
         </div>
         { (tag || footer) ?
           <div className={styles.cardRight}>
-            { tag ? <div title={tag}><ButtonMain customStyle={{ color: tagColor, cursor: 'default', background: 'white' }} label={tag} variant="outlined" /></div> : null }
+            { tag ? <div title={tag}><ButtonMain variant="outlined" size="default" theme="text" disabled customStyle={{ color: tagColor }} label={tag} /></div> : null }
```

Comment:
I wanted to see what this looked like so I added `Card` to the sandbox. I was a little confused by the `disabled` here but it looks like it is truly a `disabled` button and we should have had it originally here.